src/hotspot/share/prims/jvmtiTagMap.cpp

```diff
@@ -447,7 +436,7 @@ ClassFieldMap::ClassFieldMap() {
 }
 
 ClassFieldMap::~ClassFieldMap() {
-  for (int i=0; i<_fields->length(); i++) {
+  for (int i = 0; i < _fields->length(); i++) {
     delete _fields->at(i);
   }
   delete _fields;
```

Comment:
The changes here to ClassFieldMap are not part of the change, can you revert them (also leave mtInternal even though that's wrong).